index.html

```diff
@@ -207,6 +207,7 @@
           <p>
               Contains LPS Intellectual Property Crown copyright and database right [2015]. This information is licensed under the terms of the Open Government Licence.
           </p>
+          <br/>
```

Comment:
Do we need this br?  Can't we just increase the bottom margin on all the `p` tags in the license?  If it's used to delineate the population paragraph from the preceding ones then maybe we can put in a header or something instead?